src/widgets/health-monitoring/cluster/HealthMonitoringClusterTile.tsx

```diff
@@ -0,0 +1,169 @@
+    <Stack h="100%">
+      <SummaryHeader cpu={cpuPercent} memory={memPercent} include={properties.summary} />
+      <Accordion
+        variant="contained"
+        chevronPosition="right"
+        defaultValue={properties.defaultViewState}
+      >
+        <ResourceType
+          item={{
+            data: data.nodes,
+            icon: IconServer,
+            title: t('cluster.accordion.title.nodes'),
+            count: activeNodes,
+            length: data.nodes.length,
+            indicatorColorControl: properties.sectionIndicatorColor,
+          }}
+          id={'node'}
+          include={properties.showNode}
+          tableConfig={{ showCpu: true, showRam: true, showNode: false }}
+        />
+        <ResourceType
+          item={{
+            data: data.vms,
+            icon: IconDeviceLaptop,
+            title: t('cluster.accordion.title.vms'),
+            count: activeVMs,
+            length: data.vms.length,
+            indicatorColorControl: properties.sectionIndicatorColor,
+          }}
+          id={'vm'}
+          include={properties.showVM}
+          tableConfig={{ showCpu: true, showRam: true, showNode: false }}
+        />
+        <ResourceType
+          item={{
+            data: data.lxcs,
+            icon: IconCube,
+            title: t('cluster.accordion.title.lxcs'),
+            count: activeLXCs,
+            length: data.lxcs.length,
+            indicatorColorControl: properties.sectionIndicatorColor,
+          }}
+          id={'lxc'}
+          include={properties.showLXCs}
+          tableConfig={{ showCpu: true, showRam: true, showNode: false }}
+        />
+        <ResourceType
+          item={{
+            data: data.storage,
+            icon: IconDatabase,
+            title: t('cluster.accordion.title.storage'),
+            count: activeStorage,
+            length: data.storage.length,
+            indicatorColorControl: properties.sectionIndicatorColor,
+          }}
+          id={'storage'}
+          include={properties.showStorage}
+          tableConfig={{ showCpu: false, showRam: false, showNode: true }}
+        />
+      </Accordion>
+    </Stack>
+  );
+};
+
+interface SummaryHeaderProps {
+  cpu: number;
+  memory: number;
+  include: boolean;
+}
+
+const SummaryHeader = ({ cpu, memory, include }: SummaryHeaderProps) => {
+  const { t } = useTranslation('modules/health-monitoring');
+  if (!include) {
+    return null;
+  }
+  return (
+    <Center>
+      <Group noWrap>
+        <Flex direction="row">
+          <RingProgress
+            roundCaps
+            size={60}
+            thickness={6}
+            label={
+              <Center>
+                <IconCpu />
+              </Center>
+            }
+            sections={[{ value: cpu, color: cpu > 75 ? 'orange' : 'green' }]}
+          />
+          <Stack align="center" justify="center" spacing={0}>
+            <Text>{t('cluster.summary.cpu')}</Text>
+            <Text>{cpu.toFixed(1)}%</Text>
```

Comment:
``In [Arabic](https://en.wikipedia.org/wiki/Arabic_language), the percent sign follows the number; as Arabic is written from [right to left](https://en.wikipedia.org/wiki/Right_to_left), this means that the percent sign is to the left of the number, usually without a space.``

https://en.wikipedia.org/wiki/Percent_sign

Reply:
Good to know.

I feel like that should be handle at a global level vs per module. I see the same issue in a few other places as well (torrents widget, dns hole summary widget, etc). 

Maybe modify the existing formatPercentage() method to provide translation using an entry in the 'common' or 'tools' module. Any opposition to that?

Reply:
I wouldn't write an extra function for that since it doesn't add much value to wrap a simple translation function. I also dislike namespaces like ``common`` and ``tools`` due to  their generic nature. I think we'll leave it "untranslated" for now and implement a proper solution since we are refactoring code for the new breaking 1.0 update soon anway.

Reply:
Gotta rewrite my comment because it wasnt sent.

rtl and ltr isn't our only concern when it comes to ordering the numbers and signs that accompanies them.
We were made aware of this mostly thanks to Chinese but other Asian languages might be subject to the same problem.
Making a function for it would be overly complicated and would remove the power to correct it easily from the hands of the ones that do know better: the translators.
Since the issue was only risen recently, we've only made the change were it was specifically requested and made to be added in new PR's, but we will take it into consideration for v1.
Our benevolent translators have proven to be adapting to this really well until now so I don't think it is too much for them to handle.